daq/logger.py

```diff
@@ -44,14 +55,18 @@ def get_logger(name=None):
             LOGGERS[name].addHandler(logging.StreamHandler(sys.stdout))
         if name and set_stackdriver_client.stackdriver_handler:
             LOGGERS[name].addHandler(set_stackdriver_client.stackdriver_handler)
+
+    if log_file:
+        _add_file_handler(LOGGERS[name], log_file)
+
     return LOGGERS[name]
 
 
-def set_config(level='info', fmt=None, datefmt=None):
+def set_config(level='info'):
     """Sets config for all loggers"""
     root_logger = logging.getLogger()
     root_logger.setLevel(level)
-    formatter = logging.Formatter(fmt=fmt, datefmt=datefmt)
+    formatter = logging.Formatter(fmt=_LOG_FORMAT, datefmt=_DATE_FORMAT)
```

Comment:
This `formatter` can be replaced with _FORMATTER

Reply:
actually why is _FORMATTER not using the date format?

Reply:
consolidated